sep_021.md

```diff
@@ -103,7 +110,8 @@ This proposal does not affect backwards compatibility.
 
 Discussion <a name='discussion'></a>
 -----------------
-  
+Example 2 shows a simple representation of an Experiment that characterizes an Interlab part. The Experiment is derived from an Interlab construct, defined as an Implementation. The Experiment itself was generated by an activity. The Experiment has ExperimentalData, which is raw characterization data. Finally, two CSV files are attached to ExperimentalData: one for the absorbance values, and one for the fluorescence values.
```

Comment:
This section should be attached to the example, not in the community discussion summary.